src/ontology/uberon-edit.obo

```diff
@@ -219746,11 +219746,9 @@ is_a: UBERON:6007248 ! insect chorionic specialization
 
 [Term]
 id: UBERON:6000094
-name: clypeo-labral anlage in statu nascendi
-def: "." [FBC:VH]
-synonym: "Asn Cly" RELATED [FBC:DOS]
-is_a: UBERON:6005413 ! insect anlage in statu nascendi
-relationship: part_of FBbt:01000119
+name: obsolete clypeo-labral anlage in statu nascendi
```

Comment:
not 100% sure how this will affect mapping component? or if mapping is sufficient that you don't need to put a 'item replaced by' annotation (personally I would still add it, pointing to the FBbt term)

Reply:
Even if the mapping between the (now obsolete) Uberon term and the corresponding FBbt term still exists (because Uberon/FBbt mappings are managed on the FBbt side, so the mapping will exist until I update FBbt accordingly, and until those mappings are refreshed in Uberon), obsolete terms are ignored when generating the bridge files.